server.py

```diff
@@ -105,17 +107,9 @@ def compare_talks(talk1_id, talk2_id):
     duration2.append(talk2.duration_min)
     duration2.append(talk2.duration_sec)
     if talk1 and talk2:
+        # Reason for redundancy?
         return jsonify({"status": "success",
-                        "talk_name1": talk1.talk_name,
-                        "num_comments1": (f"{talk1.num_comments:,d}"),
-                        "num_views1": (f"{talk1.num_views:,d}"),
-                        "duration1": duration1,
-                        "talk_name2": talk2.talk_name,
-                        "num_comments2": (f"{talk2.num_comments:,d}"),
-                        "num_views2": (f"{talk2.num_views:,d}"),
-                        "duration2": duration2,
-                        "rating_list" : rating_list
-        })
+                        "talks": [talk1, talk2]})

```

Comment:
You can return the talks directly, which is far preferable to this type of redundancy.